korgan/text_integrity.py

```
@@ -53,14 +39,28 @@
         r"\s+[,;:]",
         "пробел перед знаком препинания",
     ),
+    (
+        "internal_serialization_field",
+        r"(?i)\b(?:claim_amount|case_amount|case_context|document_type|text_summary|"
+        r"verification_notes|source_urls|missing_or_unclear|important_facts|extracted_document)\b",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)